src/types/apiData.d.ts

```diff
@@ -56,12 +56,19 @@ export interface ResponseShoppingList {
  *
  */
 
+export interface RequestShoppingListItem {
+  description: string
+  quantity: number
+  unit_weight?: number
```

Comment:
This should be `number | null`, just like in the `ResponseShoppingListItem` type.